provider/bluecat/bluecat.go

```diff
@@ -1008,3 +1045,24 @@ func expandZone(zone string) string {
 	}
 	return ze
 }
+
+//Function to parse the owner from the TXT record
+//It returns in the format externaldns/owner:"OWNER_NAME"
```

Comment:
Let use the Go recommendations for function comments here.

https://golang.org/doc/effective_go#commentary

Reply:
Comment Changed to reflect recommended Golang function comments.